core/src/main/scala/caliban/execution/Executor.scala

```diff
@@ -107,10 +108,10 @@ object Executor {
 
       def reduceListStep(steps: List[Step[R]]) = {
         var i         = 0
-        val lb        = List.newBuilder[ReducedStep[R]]
+        val lb        = ListBuffer.empty[ReducedStep[R]]
```

Comment:
Aren't they the same thing?

Reply:
Yes, although the ListBuffer is the specific implementation, whereas List.newBuilder returns an instance of GrowableBuilder. It's a micro-optimization so that subsequent `addOne` calls are called on the concrete class rather than on the interface